payment/controllers_test.go

```diff
@@ -650,5 +650,9 @@ func (suite *ControllersTestSuite) TestAnonymousCardE2E() {
 
 	suite.Assert().Equal(ve.Type, vote.Type)
 	suite.Assert().Equal(ve.Channel, vote.Channel)
-	suite.Assert().Equal(ve.VoteTally, vote.VoteTally)
+	// should be number of credentials for the vote
+	suite.Assert().Equal(ve.VoteTally, int64(len(voteReq.Credentials)))
+	//issuerName := "brave.com"
```

Comment:
nit: can we delete this coment?

Reply:
incorporated: https://github.com/brave-intl/bat-go/pull/303/commits/deffea037791db48c2631fe6ff4c5167c677ac49